pkg/parse/parse.go

```diff
@@ -39,6 +39,12 @@ func ValidateVolumeOpts(options []string) ([]string, error) {
 			continue
 		}
 
+		if strings.HasPrefix(opt, "subpath") {
+			foundSubpath++
+			finalOpts = append(finalOpts, opt)
```

Comment:
Can we validate the path?  Should we run a filepath.Clean or filepath.EvalSymlinks on it to make sure it stays within the volume?